layers/+lang/javascript/packages.el

```diff
@@ -102,7 +102,7 @@
     :defer t
     :init
     (progn
-      (add-to-list 'auto-mode-alist '("\\.js\\'" . js2-mode))
+      (add-to-list 'auto-mode-alist '("\\.[m]*js\\'" . js2-mode))
```

Comment:
Unless `mmjs` is meant to be valid, this should probably be `m?js`

Reply:
one m is probably sufficient